plugins/inputs/system/net.go

```diff
@@ -80,8 +82,44 @@ func (s *NetIOStats) Gather(acc telegraf.Accumulator) error {
 			"err_out":      io.Errout,
 			"drop_in":      io.Dropin,
 			"drop_out":     io.Dropout,
+			"operational":  operational,
+			"link_mtu":     iface.MTU,
```

Comment:
It seems that somebody beat me to making this adjustment, so I'll modify the code to use the iface variable instead of using net.InterfaceByName directly, as telegraf looks to now populate this with information about all interfaces once per Gather now.